ibis/backends/pandas/tests/execution/conftest.py

```diff
@@ -62,9 +63,17 @@ def df():
             .dt.tz_localize('UTC')
             .astype(str),
             'decimal': list(map(decimal.Decimal, ['1.0', '2', '3.234'])),
-            'array_of_float64': [[1.0, 2.0], [3.0], []],
-            'array_of_int64': [[1, 2], [], [3]],
-            'array_of_strings': [['a', 'b'], [], ['c']],
+            'array_of_float64': [
+                np.array([1.0, 2.0]),
+                np.array([3.0]),
+                np.array([]),
+            ],
+            'array_of_int64': [np.array([1, 2]), np.array([]), np.array([3])],
+            'array_of_strings': [
+                np.array(['a', 'b']),
+                np.array([]),
```

Comment:
this empty array has a different dtype is that on purpose? is this correcty inferred?

Reply:
Type inference isn't tested for these arrays in particular (the Ibis types for this test DataFrame are explicitly defined later in this module)

However in general, the inferred Ibis type of a `pd.Series` that contains `np.arrays` would depend on the first element of the `pd.Series` (see [this code in this PR](https://github.com/ibis-project/ibis/pull/2753/files#diff-68fbf0c76ac970751631836a108f8af0eb8cd158f933ce27cda0e9e15f755489R137))*. This array in particular would be correctly inferred to be string dtype, since the first element is a `np.array` containing strings.

*not a foolproof way to infer the type of the column, but avoids having to check every element in the `pd.Series`